daemon/heartbeat.c

```diff
@@ -193,12 +193,12 @@ static struct {
 //
 
 static struct {
-    MTC_S32             socket;
-    struct sockaddr     sa_to[MAX_HOST_NUM];
-    WATCHDOG_HANDLE     watchdog;
-    MTC_BOOLEAN         terminate;
-    pthread_spinlock_t  lock;
-    MTC_U32             sequence[MAX_HOST_NUM];
+    MTC_S32                 socket;
```

Comment:
Some of the changes here are just white space. Can we avoid that?

Reply:
We wanted to keep the alignements of the field names because we thought it was the style convention for this project but we can remove the extra whitespaces if you'd prefer.

Reply:
Yes, we want to keep the alignment. Are these differences caused by tabs versus spaces?

Reply:
No, it's because we changed a `sockaddr` by `sockaddr_storage` which shift `sa_to` a bit further on the line and therefore we needed extra spaces for all other fields.

Reply:
Fine. I was just getting confused how this is shown here.